building_energy_standards_data/database_files/hvac_minimum_requirement_heat_rejection_90_1.json

```diff
@@ -3,270 +3,638 @@
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "Open Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 40.2,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "Closed Cooling Tower",
+        "fan_type": "Centrifugal",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 7,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "Closed Cooling Tower",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": 16.1,
+        "minimum_performance_btu_per_hr_per_hp": null,
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "Evaporative Condenser",
+        "fan_type": "Centrifugal",
+        "test_fluid": "R-448A",
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": null,
+        "minimum_performance_btu_per_hr_per_hp": 137000,
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "Evaporative Condenser",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": "R-448A",
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": null,
+        "minimum_performance_btu_per_hr_per_hp": 160000,
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "Evaporative Condenser",
+        "fan_type": "Centrifugal",
+        "test_fluid": "Ammonia",
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": null,
+        "minimum_performance_btu_per_hr_per_hp": 110000,
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "Evaporative Condenser",
+        "fan_type": "Propeller or Axial",
+        "test_fluid": "Ammonia",
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_performance_gpm_per_hp": null,
+        "minimum_performance_btu_per_hr_per_hp": 134000,
+        "annotation": "From 90.1-2022 Table 6.8.1-7"
+    },
+    {
+        "template": "90.1-2022",
+        "equipment_type": "AirCooled Condensers",
```

Comment:
"Air Cooled Condensers"?

Reply:
Edited to "Air Cooled Condensers". Thanks!